packages/crypto/tsup.config.bundled_xclud6u4tvq.mjs

```diff
@@ -0,0 +1,20 @@
+// tsup.config.ts
+import { index } from "@internal/tsup";
+var keystoreOptions = [
+  {
+    ...index,
+    entry: ["src/index.ts"],
+    format: ["cjs"]
+  },
+  {
+    ...index,
+    entry: ["src/index.ts", "src/index.browser.ts"],
+    format: ["esm"]
+  }
+];
+var tsup_config_default = keystoreOptions;
+export {
+  tsup_config_default as default,
+  keystoreOptions
+};
+//# sourceMappingURL=data:application/json;base64,ewogICJ2ZXJzaW9uIjogMywKICAic291cmNlcyI6IFsidHN1cC5jb25maWcudHMiXSwKICAic291cmNlc0NvbnRlbnQiOiBbImNvbnN0IF9faW5qZWN0ZWRfZmlsZW5hbWVfXyA9IFwiL1VzZXJzL2RhbmllbC9Eb2N1bWVudHMvRGV2ZWxvcG1lbnQvRnVlbC90cy9wYWNrYWdlcy9jcnlwdG8vdHN1cC5jb25maWcudHNcIjtjb25zdCBfX2luamVjdGVkX2Rpcm5hbWVfXyA9IFwiL1VzZXJzL2RhbmllbC9Eb2N1bWVudHMvRGV2ZWxvcG1lbnQvRnVlbC90cy9wYWNrYWdlcy9jcnlwdG9cIjtjb25zdCBfX2luamVjdGVkX2ltcG9ydF9tZXRhX3VybF9fID0gXCJmaWxlOi8vL1VzZXJzL2RhbmllbC9Eb2N1bWVudHMvRGV2ZWxvcG1lbnQvRnVlbC90cy9wYWNrYWdlcy9jcnlwdG8vdHN1cC5jb25maWcudHNcIjtpbXBvcnQgeyBpbmRleCB9IGZyb20gJ0BpbnRlcm5hbC90c3VwJztcbmltcG9ydCB0eXBlIHsgT3B0aW9ucyB9IGZyb20gJ3RzdXAnO1xuXG5leHBvcnQgY29uc3Qga2V5c3RvcmVPcHRpb25zOiBPcHRpb25zW10gPSBbXG4gIHtcbiAgICAuLi5pbmRleCxcbiAgICBlbnRyeTogWydzcmMvaW5kZXgudHMnXSxcbiAgICBmb3JtYXQ6IFsnY2pzJ10sXG4gIH0sXG4gIHtcbiAgICAuLi5pbmRleCxcbiAgICBlbnRyeTogWydzcmMvaW5kZXgudHMnLCAnc3JjL2luZGV4LmJyb3dzZXIudHMnXSxcbiAgICBmb3JtYXQ6IFsnZXNtJ10sXG4gIH0sXG5dO1xuXG5leHBvcnQgZGVmYXVsdCBrZXlzdG9yZU9wdGlvbnM7XG4iXSwKICAibWFwcGluZ3MiOiAiO0FBQStULFNBQVMsYUFBYTtBQUc5VSxJQUFNLGtCQUE2QjtBQUFBLEVBQ3hDO0FBQUEsSUFDRSxHQUFHO0FBQUEsSUFDSCxPQUFPLENBQUMsY0FBYztBQUFBLElBQ3RCLFFBQVEsQ0FBQyxLQUFLO0FBQUEsRUFDaEI7QUFBQSxFQUNBO0FBQUEsSUFDRSxHQUFHO0FBQUEsSUFDSCxPQUFPLENBQUMsZ0JBQWdCLHNCQUFzQjtBQUFBLElBQzlDLFFBQVEsQ0FBQyxLQUFLO0FBQUEsRUFDaEI7QUFDRjtBQUVBLElBQU8sc0JBQVE7IiwKICAibmFtZXMiOiBbXQp9Cg==
```

Comment:
Was this file added by mistake?

Reply:
Indeed

Reply:
[b1d6458](https://github.com/FuelLabs/fuels-ts/pull/1775/commits/b1d645804d2d33b801a3d743a9dd90ff89d11224)

Reply:
#1779